### ISSUE TITLE
VSCode copy output feature

### ISSUE THREAD

**Issue by smcenlly** [wallabyjs/quokka#238]
### Feature Request
I would like a feature to be able to copy the entire output from the VSCode Quokka output window to the clipboard. I would also like to be able to copy a single output section to the clipboard. 

These should be available as both VSCode Commands and as links from the VSCode output window.

**Comment by myleftshoe** [comment]
Would like to see this too. I recenty posted a question on stackoverflow where it would have been handy. I was testing some code using quokka with the expected output displayed alongside the code. I wanted to paste this directly into the SO question but instead had to post the code only, then copy the expected output line by line from the console window and paste it next to the relevant line in the code - a lot more time consuming and error prone.

**Comment by rdhar** [comment]
This looks like another great consideration to push forward #66. Huge quality-of-life improvement to copy the entire output as it appears, both for sharing and documentation.

**Comment by DmitriyMalayev** [comment]
This can probably be achieved by using a screenshot tool which selects code/text from image.

**Comment by empz** [comment]
Really need this.

**Comment by ArtemGovorov** [comment]
@empz Have you tried `Quokka.js: Copy Value` command (`Cmd/Ctrl + K, X`)? It uses current editor context and will evaluate/copy expressions/variables under cursor or just selected in editor.

**Comment by ArtemGovorov** [comment]
@empz `Copy Value` has the following limitations at the moment: `maximum of 5000 elements in an array / object properties, maximum depth of 10, maximum single string length of 8192 characters.` I guess in your case the limits are exceeded, is it correct? 

While the existing limits are pretty deep to accomodate most of use cases, we can make them configurable so that in your case you will be able to specify how deep/far you'd like the objects to be copied. Will it work for you?

**Comment by empz** [comment]
Yes, the limits are exceeded. A configurable option would be useful.

**Comment by ArtemGovorov** [comment]
@empz Mind sharing what exact limits are you hitting in your case? Is it max number of an array items, max number of an object properties, max traversal depth, or max string length?
